packages/connector-edge/appveyor.yml

```diff
@@ -0,0 +1,19 @@
+environment:
```

Comment:
Does this file have any effect?

Reply:
I was using `appveyor` to verify it at least compiles. The problem we are going to have is that we have a dependency on Windows to download the `edge-diagnostics-adapter` so I'm not sure how that's going to play out with travis 😞

Reply:
I will remove this file for the time being.